.github/workflows/main.yml

```diff
@@ -15,7 +15,15 @@ jobs:
         jdk: [ 8, 11 ]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Pull Request Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
Instead of checking out from opprop, we should check out from head repo. This will make `CI_ORGANIZATION` equal to the head repo.

Reply:
This is copied from https://github.com/opprop/checker-framework-inference/blob/3554b916cc2c8723086c4c3f6857d01829ad1fdb/.github/workflows/main.yml#L24